docs/managing_deis/upgrading-deis.rst

```diff
@@ -68,6 +68,77 @@ Upgrade Deis clients
 As well as upgrading ``deisctl``, make sure to upgrade the :ref:`deis client <install-client>` to
 match the new version of Deis.
 
+Graceful Upgrade
+----------------
+
+Alternatively, an experimental feature exists to provide the ability to perform a graceful upgrade. This process is
+available for version 1.9.0 moving foward and is intended to facilitate upgrades within a major version - upgrading
+between major versions is not supported. Unlike the in-place process above, this process keeps the platform's
+routers and publishers up during the upgrade process. This means that there should only be a maximum of around 1-2
+seconds of downtime while the routers boot up. Many times, there will be no downtime at all.
+
+.. note::
+
+    Your loadbalancer configuration is the determining factor for how much downtime will occur during a successful upgrade.
+    If your loadbalancer is configured to quickly reactivate failed hosts to its pool of active hosts, its quite possible to
+    achieve zero downtime upgrades. If your loadbalancer is configured to be more pessimistic, such as requiring multiple
+    successful healthchecks before reactiving a node, then the chance for downtime increases. You should review your
+    loadbalancers configuration to determine what to expect during the upgrade process.
+
+The process involves two ``deisctl`` subcommands, ``upgrade-prep`` and ``upgrade-takeover``, in coordination with a few other important commands.
+
+First, a new ``deisctl`` version should be installed to a temporary location, reflecting the desired version to upgrade
+to. Care should be taken not to overwrite the existing ``deisctl`` version.
+
+.. code-block:: console
+
+    $ mkdir /tmp/upgrade
+    $ curl -sSL http://deis.io/deisctl/install.sh | sh -s 1.10.0 /tmp/upgrade
+    $ /tmp/upgrade/deisctl --version  # should match the desired platform
+    1.10.0
+    $ /tmp/upgrade/deisctl refresh-units
+    $ /tmp/upgrade/deisctl config platform set version=v1.10.0
+
+.. note::
```

Comment:
While this note is accurate, I think it could be rewritten to be more conversational. Something like "For example: to upgrade from Deis v1.9.0 to v1.10.0...". I think users can understand what would need to be swapped out for various versions.